include/deal.II/base/polynomials_nedelec_sz.h

```diff
@@ -0,0 +1,65 @@
+// ---------------------------------------------------------------------
```

Comment:
since the class is called `IntegratedLegendreSZ`, maybe call the file `polynomials_integrated_legendre_sz.h` (and similar for the .cc file)?